DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/sanity/test_cleanup_fixture.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+pytestmark = pytest.mark.suite_my
+
+@pytest.fixture
+def add_cleanup():
```

Comment:
Have you looked at utils.tb_utils:tb_clean_config_device() ?

Reply:
`clean_config_device` is great for some device configurations, but not for all. For example, it doesn't clean `dcb` config (which we would need later). Although we can expand the functionality since `clean_config_device` is used only in one other test.
There is another issue: when we call `tgen_utils_connect_to_tgen` we are creating a new session on Ixia and if we don't call `tgen_utils_stop_protocols` then the session will be left hanging. So, we need some sort of cleanup functionality anyway.